stages/org.osbuild.systemd

```diff
@@ -73,6 +87,21 @@ def main(tree, options):
     if default_target:
         subprocess.run(["systemctl", "--root", tree, "set-default", default_target], check=True)
 
+    if presets:
+        svcs = {}
+
+        for svc in presets:
+            # verify that we don't have duplicate service names with different
+            # states
+            if svc["name"] in svcs:
+                raise RuntimeError(f"{svc['name']!r} has multiple defined states.")
```

Comment:
Since the order doesn't matter, we could change the schema so that the data comes in already as key value pairs (`"name": "state"`) and it would save us from this issue.

Reply:
Unless the order does matter?  Maybe with wildcards?

Reply:
yeah, I think order does matter https://www.freedesktop.org/software/systemd/man/systemd.preset.html#Examples

```
Example 4. Administrator policy

# /etc/systemd/system-preset/00-lennart.preset

enable httpd.service
enable sshd.service
enable postfix.service
disable *

This enables three specific services and disables all others. This is useful for administrators to specifically select the units to enable, and disable all others. Due to the filename prefix "00-" it will be read early and override all other preset policy files.
```

Reply:
Right, all good then.